types/three/three-core.d.ts

```diff
@@ -5329,6 +5329,11 @@ export interface WebGLRendererParameters {
      * default is false.
      */
     logarithmicDepthBuffer?: boolean;
+
+	/**
+     * default is null.
+     */
+    context?: CanvasRenderingContext2D | WebGLRenderingContext;
```

Comment:
I'm not sure that `CanvasRenderingContext2D` would be valid, can you explain why you allow it here?

Reply:
Yeah, `CanvasRenderingContext2D` is only valid for the `CanvasRenderer` class, which is now deprecated, and removed from core.

Otherwise, `WebGLRenderingContext` on its own is valid.